### PR TITLE
feat(cli/coverage): print lines with no coverage to stdout

### PR DESCRIPTION
**Pull request by caspervonb**
This pretty prints uncovered lines as red text with line numbers to stdout with the coverage report.

Screenshot 2020-09-23 at 2 18 39 PM

### CODE REVIEW

File: cli/coverage.rs [hunk 1]
```diff
@@ -277,48 +265,57 @@ impl PrettyCoverageReporter {
...
+            lines[line_index],
+            width = 4
+        );
+
+        println!("{}", colors::red(&line));
+      }
+    }
   }
+
+  pub fn close(&self) {}
```
Comment (caspervonb) [inline cli/coverage.rs:318]: Final summary of all coverage will (eventually) go here; just haven't figured out what that looks like yet.

File: cli/coverage.rs [hunk 2]
```diff
@@ -277,48 +265,53 @@ impl PrettyCoverageReporter {
...
+      for line_index in uncovered_lines {
+        let line =
+          format!("{:width$} {}", line_index + 1, lines[line_index], width = 4);
+
+        println!("{}", colors::red(&line));
+      }
+    }
   }
+
+  pub fn close(&self) {}
```
Comment (caspervonb) [inline cli/coverage.rs:314]:
```suggestion
```

File: cli/main.rs [hunk 3]
```diff
@@ -601,19 +601,18 @@ async fn test_command(
...
+    let mut coverage_reporter = PrettyCoverageReporter::new(quiet);
+    for coverage in filtered_coverages {
+      coverage_reporter.visit_coverage(&coverage);
+    }
 
-    let pretty_coverage_reporter =
-      PrettyCoverageReporter::new(filtered_coverage);
-    let report = pretty_coverage_reporter.get_report();
-    print!("{}", report)
+    coverage_reporter.close();
```
Comment (caspervonb) [inline cli/main.rs:615]:
```suggestion
```

**Review by ry — APPROVED** [review]
Nice

**Comment by ry** [comment]
Is coverage an unstable feature? Can we land this immediately?

**Comment by caspervonb** [comment]
Yeah its unstable 👍

**Pull request merged**